docs/studios/unity/unity-sdk/in-editor-replay.md

```diff
@@ -4,10 +4,61 @@ sidebar_label: 'Bot Replay'
 
 # Replaying Bot Behaviors in the Unity Editor
 
-Bots running on Regression Games save all states observed and actions sent, which can be used in the editor
-for playback. This feature, combined with our validation framework, allows you to find and debug
-specific errors within your game.
+Bots running on Regression Games save all states observed, all actions sent, and all validations performed.  This data can be loaded in the editor for visualization and playback in the scene. This feature, combined with our validation framework, allows you to find and debug specific errors within your game.
 
-## The Bot Replay Pane
+## Downloading the Replay Data
 
-![timeline panel](timeline_panel.png)
\ No newline at end of file
+At the end of a Bot's lifecycle, the replay data for every game tick processed by that bot is saved into a single Zip file.  This file can be downloaded from the `History` section of the `Running Bots` tab in the Regression Games client.  Click the `Download Replay Data` button for the Bot instance you want to download.  This will download a file named `rg_bot_replay_data-<botInstanceId>.zip`.
```

Comment:
Since `botInstanceId` is presented in the client as simply `instance ID`, I'd go with that instead

Reply:
resolved